src/shared/helpers/getCourseDate.ts

```diff
@@ -0,0 +1,6 @@
+export const getCourseDate = (startDate: string) => {
+  const fourteenDays = 14 * 24 * 60 * 60 * 1000;
+  const isStale = Date.parse(startDate) < Date.now() - fourteenDays;
+
+  return isStale ? 'TBD' : startDate;
+};
```

Comment:
With the current implementation, I can pass an empty string or a random string like 'brah', and the function will accept and return it. I'd recommend adding some validation to prevent this. 
Additionally, I'm wondering if we should change the date format in `courses.data.ts` to ISO 8601 format